src/libraries/System.Runtime/ref/System.Runtime.cs

```diff
@@ -13955,6 +13955,11 @@ public sealed partial class HandleProcessCorruptedStateExceptionsAttribute : Sys
     {
         public HandleProcessCorruptedStateExceptionsAttribute() { }
     }
+    public delegate bool UnhandledExceptionHandler(System.Exception exception);
+    public static partial class ExceptionHandling
+    {
+        public static void SetUnhandledExceptionHandler(UnhandledExceptionHandler handler) { throw null; }
```

Comment:
You may want to run it through https://github.com/dotnet/runtime/blob/main/docs/coding-guidelines/updating-ref-source.md to make sure that the formatting is correct.

Reply:
The document does not say much about formatting. It looks like the binary/IL shape is most important for ref assemblies. 
I did try to follow the same patterns as existing code in this file.

Reply:
The document has instructions how to rerun the ref-assembly generation tool that reformats the file as side-effect. I have done that and push the formatting change (together with formatting diffs introduced by others) to this PR.

Reply:
Thanks!! Somehow I thought that I've done that step and it resulted in no changes in ref assembly sources. Perhaps I did that in a wrong enlistment...